apps/events/urls.py

```diff
@@ -48,14 +48,3 @@
 # API v1
 router = SharedAPIRootRouter()
 router.register("events", views.EventViewSet, basename="events")
-router.register("attendees", views.AttendeeViewSet, basename="attendees")
-router.register(
-    prefix="registration/attendance-events",
-    viewset=views.RegistrationAttendaceEventViewSet,
-    basename="registration_attendance_events",
-)
-router.register(
-    prefix="registration/user-attendees",
-    viewset=views.RegistrationAttendeeViewSet,
-    basename="registration_attendees",
-)
```

Comment:
These were mostly not used at all, maybe just payments in OWF. That can be fixed easily

Reply:
Can you test if OWF use them? Could be nice since OW4 relies on OWF for payments.
If it's too much hassle, there's no events with payments or Nibble to update balance for, so this is a good time to test live 👼

Reply:
OWF uses them for event payments, but I have a PR on fixing things to this API already :smile: